qa/suites/basic/health-ok.sh

```diff
@@ -68,7 +67,8 @@ run_stage_3 "$CLI"
 ceph_cluster_status
 ceph_health_test
 ceph_log_grep_enoent_eaccess
-systemd_ceph_osd_target_wants
+test_systemd_ceph_osd_target_wants
```

Comment:
Can this ever run on a host that has no OSDs? (In which case it will fail)

Reply:
Another good catch. Fixing.